src/requested-data/requested-data-models.ts

```diff
@@ -38,3 +40,55 @@ export interface Result {
     partyCode: string;
     partyName: string;
 }
+
+// New models
```

Comment:
Unnecessary comment (comments need to maintained just like code).

Reply:
It was intended to make it easier to remove when it is deprecated by finishing the upgrade to the new data structure. But we can find it through the lack of references.
Removed the comment.